src/App.js

```diff
@@ -315,10 +266,15 @@ const App = () => {
     return WEEKDAYS.indexOf(firstDay) - WEEKDAYS.indexOf(secondDay)
   }
 
-  const renderPickupsByDayOfTheWeek = (weekday) => {
-    const pickupsHappeningOnDay = PICKUPS.filter(pickup => (pickup.day === weekday) && !pickup.hidden)
-    if (pickupsHappeningOnDay.length === 0) return <p>No pickups</p>
-    return pickupsHappeningOnDay.map(pickup => <PickupCard field={pickup.field} address={pickup.address} time={pickup.time} day={pickup.day} contact={pickup.contact}/>)
+  const renderPickupsByDayOfTheWeek = (weekday, search) => {
+    if (weekday && search) { 
+      const pickupsHappeningOnDay = searchResults.filter(pickup => (pickup.day === weekday) && !pickup.hidden);
+      return pickupsHappeningOnDay.map(pickup => <PickupCard field={pickup.field} address={pickup.address} time={pickup.time} day={pickup.day} contact={pickup.contact}/>)
+    } else { 
```

Comment:
You can move this logic into its own function as `renderSearchResultsByDayOfWeek`.